scd30/scd30.c

```diff
@@ -183,6 +185,18 @@ int16_t scd30_set_forced_recalibration(uint16_t co2_ppm) {
     return ret;
 }
 
+int16_t scd30_read_serial(char *serial) {
+    int16_t ret;
+
+    ret = sensirion_i2c_write_cmd(SCD30_I2C_ADDRESS, SCD30_CMD_READ_SERIAL);
+    if (ret)
+        return ret;
+
+    sensirion_sleep_usec(SCD30_WRITE_DELAY_US);
+    return sensirion_i2c_read_bytes(SCD30_I2C_ADDRESS, (uint8_t *)serial,
+                                    SCD30_SERIAL_NUM_WORDS);
```

Comment:
Should we enforce zero termination of the serial buffer?

Reply:
absolutely